sphinx/cmdline.py

```diff
@@ -114,7 +114,7 @@ def handle_exception(app, opts, exception, stderr=sys.stderr):
                   file=stderr)
 
 
-def main(argv):
+def main(argv=sys.argv[1:]):
```

Comment:
mypy warns here: https://travis-ci.org/sphinx-doc/sphinx/jobs/231257064#L753
Please mark this line as `# typing: ignore`.